pkg/policy/api/rule.go

```diff
@@ -336,6 +336,10 @@ type CIDRRule struct {
 	//
 	// +optional
 	ExceptCIDRs []CIDR `json:"except,omitempty"`
+
+	// Generated indicates whether the rule was generated based on other rules
+	// or provided by user
+	Generated bool `json:"generated,omitempty"`
```

Comment:
Do we really need this to be exported to json?

Reply:
@aanm It will be in JSON only if it's set to true, which I think makes sense.

Reply:
@nebril but the user will never (should never?) use this field, right?

Reply:
@aanm he should not set it, but I think he should be able to see it if it's set.